scripts/arm32_ci_script.sh

```diff
@@ -146,9 +146,8 @@ function cross_build_corefx {
     git am < "$__ARMRootfsMountPath"/dotnet/setenv/corefx_cross.patch
 
     #Cross building for emulator rootfs
-    ROOTFS_DIR="$__ARMRootfsMountPath" CPLUS_INCLUDE_PATH=$LINUX_ARM_INCPATH CXXFLAGS=$LINUX_ARM_CXXFLAGS
-    BUILD_NATIVE="$ROOTFS_DIR" ./build-native.sh -buildArch=$__buildArch -$__buildConfig -- cross $__verboseFlag
-    BUILD_MANAGED="$ROOTFS_DIR" ./build-managed.sh -buildArch=$__buildArch -$__buildConfig -skipTests
+    ROOTFS_DIR="$__ARMRootfsMountPath" CPLUS_INCLUDE_PATH=$LINUX_ARM_INCPATH CXXFLAGS=$LINUX_ARM_CXXFLAGS ./build-native.sh -buildArch=$__buildArch -$__buildConfig -- cross $__verboseFlag
+    ROOTFS_DIR="$__ARMRootfsMountPath" CPLUS_INCLUDE_PATH=$LINUX_ARM_INCPATH CXXFLAGS=$LINUX_ARM_CXXFLAGS ./build-managed.sh -buildArch=$__buildArch -$__buildConfig -skipTests
```

Comment:
I think you wrote codes to run the managed build
`... ./build-managed.sh -buildArch=$__buildArch  ...`
The first argument may be 'arm-softfp', but in 'build-managed.sh', It doesn't write for processing about 'arm-softfp'.

build-native.sh
``
21 while :; do
 22     if [ $# -le 0 ]; then
 23         break
 24     fi
 25 
 26     lowerI="$(echo $1 | awk '{print tolower($0)}')"
 27     case $lowerI in
 28         freebsd)
 29             __TestNugetRuntimeId="$__TestNugetRuntimeId=osx.10.10-x64"
 30             __BuildOS="$__BuildOS=freebsd"
 31             __TargetOS="$__TargetOS=freebsd"
 32             ;;
 33         netbsd)
 34             __TestNugetRuntimeId="$__TestNugetRuntimeId=osx.10.10-x64"
 35             __BuildOS="$__BuildOS=netbsd"
 36             __TargetOS="$__TargetOS=netbsd"
 37             ;;
 38         osx)
 39             __TestNugetRuntimeId="$__TestNugetRuntimeId=osx.10.10-x64"
 40             __BuildOS="$__BuildOS=osx"
 41             __TargetOS="$__TargetOS=osx"
 42             ;;
 43         windows)
 44             __TestNugetRuntimeId="$__TestNugetRuntimeId=win7-x64"
 45             __BuildOS="$__BuildOS=windows_nt"
 46             __TargetOS="$__TargetOS=Windows_NT"
 47             ;;
 48         useservergc)
 49             __ServerGC=1
 50            e;
 51         *)
 52             __UnprocessedBuildArgs="$__UnprocessedBuildArgs $1"
 53     esac
 54 
 55     shift
 56 done
``